gcd/src/main/java/org/spine3/server/storage/datastore/Kind.java

```diff
@@ -66,6 +81,14 @@ public static Kind of(TypeName typeName) {
         return new Kind(typeName.value());
     }
 
+    /**
+     * Produces a {@code Kind} representing the datastore namespace kind.
+     */
+    @Internal
+    public static Kind ofNamespace() {
```

Comment:
This is better than making the ancillary kind constructor `public` (or any but `private`).

I agree that semantically this may belong to `tenant` package, but this is an exceptional case when we walk around the kind validation rules. Therefore, I believe, this is the best way to prevent the user from breaking through the validation by himself.